### DISCUSSION TITLE
Disable inlay hints in Red Hat's vscode-java

### DISCUSSION THREAD

**Discussion by Vampyrr85** [redhat-developer/vscode-java#2668] [Q&A]
Using red hat adds this extra thing in and I cannot figure out how to disable it heres an example from a basic exercise

 System.out.printf("%.1f C = %.1f F", celsius, farenheit); is what I want it to look like 
 System.out.printf(format: "%.1f C = %.1f F", celsius, farenheit); is what it adds and it bothers me but I do not know where this feature is to turn it off

**Comment by fbricon** [comment]
You can disable inlay hints with this setting:

> `java.inlayHints.parameterNames.enabled`: Enable/disable inlay hints for parameter names. Supported values are: `none`(disable parameter name hints), `literals`(Enable parameter name hints only for literal arguments) and `all`(Enable parameter name hints for literal and non-literal arguments). Defaults to `literals`.
